tests/datasets/test_wms.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import requests
```

Comment:
We don't currently have a dependency on requests. Is this needed?

Reply:
possibly not, let me see as it would solve a bunch of my other issues